### PR TITLE
Feature graphic style v2

### PR DESCRIPTION
**Pull request by Kit486759**
Style for musician page and pricing page.

### CODE REVIEW

**Review by miyabitanimchi — COMMENTED** [review]
The design looks amazing! Great work Kit 🙌 
But one thing, just my opinion but I thought some font size is a bit too small
スクリーンショット 2021-11-10 午後7 58 20
<br/>
スクリーンショット 2021-11-10 午後7 59 44

File: src/components/MusicianPage/MusicianPage.js [hunk 1]
```diff
@@ -3,36 +3,44 @@ import { connect } from "react-redux";
...
-      <MusicPlayer song={user.profile.song} />
+    <div className="shelf">
+      {user.profile.occupation === "artist" ? (
+        <div className="microphone"></div>
+      ) : (
+        <div className="headphone"></div>
+      )}
+      <div className="pass-name">
+        <p>{user.profile.lastName}</p>
+        <p>{user.profile.lastName}</p>
```
[the remark this answers is not in the document]
Comment (Kit486759) [reply src/components/MusicianPage/MusicianPage.js]: Thank you Miyabi. I have fixed it 😀

**Review by BlackStone8960 — DISMISSED** [review]
It looks amazing! Thank you so much :)

**Comment by Kit486759** [comment]
> The style looks a little broken in the pictures below, so could you change them?
> 
> Other than that, it looks all good :)
> 
> 
> 
> Screen Shot 2021-11-11 at 2 02 24 PM
> 
> Screen Shot 2021-11-11 at 2 03 43 PM
> 
> 
> 
> 

@YumiKimbara thank you for this! I will fix it.

**Comment by Kit486759** [comment]
Hi @YumiKimbara . I fixed the position of name and genre.  Please review. Thank you!

**Comment by YumiKimbara** [comment]
I confirmed that it was fixed. Thank you :)

**Review by YumiKimbara — APPROVED** [review]

**Pull request merged**